config.nims

```diff
@@ -0,0 +1,51 @@
+# Nim compilation defaults for the whole reposiotry
+--gc: arc
+--checks: on
+--assertions: on
+--spellSuggest
+--styleCheck: error
+
+if defined(release) or defined(danger):
+  --opt: speed
+  --passC: "-flto"
+  --passL: "-flto"
+  --passL: "-s"
+
+import std/[os, sequtils]
+from std/strutils import startsWith, endsWith
+from std/strformat import `&`
+
+const IgnorePathPrefixes = [".", "scripts"]
+
+func isIgnored(path: string): bool =
+  IgnorePathPrefixes.mapIt(path.startsWith(it)).anyIt(it)
+
+iterator modules(): string =
+  ## Iterate over paths to all nim files in current directory, skipping
+  ## paths starting with substrings from the `IgnorePathPrefixes` const
+  for path in walkDirRec(getCurrentDir(), relative = true):
+    if not path.isIgnored() and path.endsWith(".nim"):
+      yield path
+
+############ Tasks
+task test, "Test everything":
+  --hints: off
+  var failedUnits: seq[string]
+
+  for path in modules():
+    if not path.isIgnored() and path.endsWith(".nim"):
+      echo &"Testing {path}:"
```

Comment:
This is fine for now, but I will probably want to reduce the verbosity as soon as we have more than a few dozen of scripts.

Reply:
This line just delineates the `unittest` output. With current config when everything passes smoothly outputs for different modules don't have any other separation.